qa/L0_e2e/generate_example_model.py

```diff
@@ -197,6 +197,8 @@ def serialize_model(model, directory, output_format='xgboost'):
 
 def generate_config(
         model_name,
+        *,
```

Comment:
Is this just to ensure that we don't use positional args for kwargs?

Reply:
Yes